src/components/Table/index.tsx

```diff
@@ -238,10 +251,11 @@ const Table = React.forwardRef<TableRef, TableProps>((props, ref) => {
             );
           }
         },
-        ...VList({
-          height: height, // 此值和scrollY值相同. 必传. (required).  same value for scrolly
-          vid: vid
-        })
+        ...vt
+        // ...VList({
+        //   height: height, // 此值和scrollY值相同. 必传. (required).  same value for scrolly
```

Comment:
Remove Chine symbols or the whole comment.

Reply:
done